apps/system/js/multi_screen_controller.js

```diff
@@ -10,7 +10,8 @@
     'mozChromeEvent'
```

Comment:
There must be some mistakes in the previous version. Since we've defined `mozChromeEvent` here, it shouldn't be necessary to add or remove event listeners manually. I can't remember why I did this at that time but could you do me a favor to remove all the `window.addEventListener` and `window.removeEventListener` below? Thanks a lot.